src/components/Date/Date.jsx

```diff
@@ -84,18 +84,18 @@ class Date extends React.Component {
     }
 
     return (
-      <div className={!isValid ? 'input-error-date' : undefined}>
-        <label>
-          {this.props.label ? this.props.label : 'Date of birth'}
+      <fieldset className={!isValid ? 'input-error-date' : undefined}>
+        <legend className="vads-u-font-size--base vads-u-font-weight--normal">
+          {this.props.label || 'Date of birth'}
           {this.props.required && (
             <span className="form-required-span">(*Required)</span>
           )}
-        </label>
+        </legend>
         {errorSpan}
         <div
           className={isValid ? undefined : 'usa-input-error form-error-date'}
         >
-          <div className="usa-date-of-birth">
+          <div className="usa-date-of-birth usa-datefields clearfix-text">
```

Comment:
I took a look and noticed the floats weren't clearing, and that's why the error red border wasn't as tall as the inputs. If I changed "clearfix-text" to "clearfix", the floats cleared as expected.

Reply:
I'll take care of this first thing tomorrow!